dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsConnectionTestSuite.scala

```diff
@@ -11,15 +14,81 @@ import scala.concurrent.{Await, Future}
 class WsConnectionTestSuite extends WsSuiteBase {
 
   override protected val dexInitialSuiteConfig: Config = ConfigFactory
-    .parseString(s"""waves.dex.price-assets = [ "$BtcId", "WAVES" ]""")
+    .parseString(s"""waves.dex.price-assets = [ "$BtcId", "$UsdId", "WAVES" ]""")
     .withFallback(jwtPublicKeyConfig)
 
   override protected def beforeAll(): Unit = {
     wavesNode1.start()
-    broadcastAndAwait(IssueBtcTx)
+    broadcastAndAwait(IssueBtcTx, IssueUsdTx)
     dex1.start()
   }
 
+  "Matcher should send rates snapshot after initial message and rates updates" in {
+
+    def assertRatesUpdates(wsc: WsConnection)(expectedRatesUpdates: List[(Map[Asset, Double], Long)]): Unit = {
+      wsc
+        .receiveAtLeastN[WsRatesUpdates](expectedRatesUpdates.size)
+        .map(r => r.rates -> r.updateId) should matchTo(expectedRatesUpdates)
+    }
+
```

Comment:
maybe we need to check the cases:
1. rate is already in config (before the ws-connection was made)
2. asset  doesn't have the rate before we make the ws-connection

Reply:
1. At least the first test _Rates snapshot_ checks this
2. Actually this PR is about the rates changes notification, so there aren't some problems with this kind of assets